squire_lib/src/tournament.rs

```diff
@@ -918,6 +919,60 @@ impl Tournament {
         }
         digest
     }
+
+    /// Generates a round slip in HTML
+    pub fn round_slips_html(&self, css: &str) -> String {
+        let mut ret: String = "<!DOCTPYE HTML>".to_string();
+
+        let _ = write!(ret, "<html lang=\"en\">");
+        let _ = write!(ret, "<head><style>");
+        let _ = write!(ret, "{}", &css);
+        let _ = write!(ret, "</style></head>");
+        let _ = write!(ret, "<body>");
+
+        for r in self.round_reg.rounds.values().filter(|r| r.is_active()) {
+            let _ = write!(ret, "<div class='card' style='border-style: solid;'>");
+            let _ = write!(ret, "<div class='title' style='display: flex; align-items: center; width: 100%; flex-direction: column;'>");
+            let _ = write!(ret, "{}", &html_escape::encode_text(&self.name.to_string()));
+            let _ = write!(ret, "</div>");
+            let _ = write!(ret, "<table style='width: 100%;'>");
+            let _ = write!(ret, "<tr><tr><td>Round # ");
+            let _ = write!(ret, "{}", &r.match_number.to_string());
```

Comment:
Rust's format strings apply a value's string conversion, so this `.to_string()` is not needed:
``` suggestion
            let _ = write!(ret, "{}", r.match_number);
```
Note that this string conversion takes a reference to value, so the explicit reference is not needed here.